src/libsync/progressdispatcher.cpp

```diff
@@ -189,9 +189,17 @@ void ProgressInfo::adjustTotalsForFile(const SyncFileItem &item)
         return;
     }
 
-    _fileProgress._total += item._affectedItems;
-    if (isSizeDependent(item)) {
-        _sizeProgress._total += item._size;
+    // Update _currentItems map
+    setProgressItem(item, 0);
+
+    // Update _sizeProgress._total
+    _sizeProgress._total = 0;
+    QMutableHashIterator<QString, ProgressItem> it(_currentItems);
```

Comment:
very minor: Mutable?

Reply:
Done, copy/pasted loop from somewhere else in that file.